test/end2end_test.go

```diff
@@ -216,8 +217,8 @@ func (s *testServer) StreamingOutputCall(args *testpb.StreamingOutputCallRequest
 			return grpc.Errorf(codes.DataLoss, "expected an :authority metadata: %v", md)
 		}
 		// For testing purpose, returns an error if there is attached metadata except for authority.
```

Comment:
I think the purpose of this is to return an error in certain conditions, to test that the client gets the correct error.
Changed the comment to include this.